time stamp
wallpattern_detection/ObjectWithType[] objects
